src/client/components/ui/Card/Link/LinkCard.tsx

```diff
@@ -17,30 +16,33 @@ interface LinkCardProps extends Pick<React.HTMLAttributes<unknown>, 'className'>
 
 export function LinkCard({ children, className, imageUrl, link, linkLabel, title, titleAs }: React.PropsWithChildren<LinkCardProps>)  {
 
-	function LinkCardTitle({ children, className }: { titleAs?: HtmlHeadingTag } & React.HTMLAttributes<HTMLTitleElement>) {
-		return React.createElement(titleAs || 'h3', { className: className }, children);
+	function LinkCardTitle({ children, className, id }: { titleAs?: HtmlHeadingTag, id: string } & React.HTMLAttributes<HTMLTitleElement>) {
+		return React.createElement(titleAs || 'h3', { className: className, id }, children);
 	}
 
+	const idIntitule = useId();
+	const idLink = useId();
+
 	return (
-		<Link href={link} className={classNames(styles.card, 'underline-none')} prefetch={false}>
-			<article className={classNames(styles.cardArticle, className)}>
-				<div className={styles.cardImageWrapper}>
-					<Image src={imageUrl} alt="" width={328} height={180} />
+		<article className={classNames(styles.card, className)}>
+			<div className={styles.cardImageWrapper}>
+				<Image src={imageUrl} alt="" width={328} height={180} />
+			</div>
+
+			<div className={styles.cardContent}>
+				<div className={styles.cardContentHeader}>
+					<LinkCardTitle id={idLink} className={styles.cardTitle}>{title}</LinkCardTitle>
+					<span className={styles.cardAction}>
+						<span className="sr-only">{linkLabel}</span>
+						<Link id={idLink} href={link} className={classNames('underline-none')} prefetch={false}>
+							<Link.Icon aria-labelledby={`${idIntitule}, ${idLink} en savoir plus`} name='angle-right' aria-hidden="false" />
+						</Link>
```

Comment:
C'est le lien qui devrait prendre le label, pas l'icône.
```suggestion
						<Link aria-labelledby={`${idIntitule}, ${idLink}`} id={idLink} href={link} className={classNames('underline-none')} prefetch={false}>
							<Link.Icon name='angle-right' aria-hidden="false" />
						</Link>
```
Also, la partie textuelle du lien en `sr-only` devrait faire partie du lien